src/DateInput.tsx

```diff
@@ -416,7 +418,16 @@ export function DateInputTypeInField ({ name, ...rest }: InputProps) {
   }, [])
 
   return (
-    <Field name={name} validate={validate}>
+    <Field
+      name={name}
+      validate={(value: string) => {
+        if (validate) {
+          return validate(value) ? validate(value) : isValidDate(value)
+        }
+
+        return value ? isValidDate(value) : undefined
+      }}
```

Comment:
Oh you're right. But that won't work either: if `validate`, always run `isValidDate` won't work.

Maybe `isValidDate` needs to check for a `value`. If no `value`, return undefined, because it's an empty field.

Reply:
🤦‍♀️ thank you! yes, I think that is right now because it'll allow the custom `validate` to determine whether this field is required or not :)